src/services/notehub/AxiosHttpNotehubAccessor.ts

```diff
@@ -90,6 +87,14 @@ export default class AxiosHttpNotehubAccessor implements NotehubAccessor {
     return getError(errorCode, { cause: e as Error });
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  constructStartDate(startDate: number) {
```

Comment:
I see your point and I've tried to address it. I renamed the function to `getEpochChartDataDate` and moved it into the `uiHelpers.ts` file (because I couldn't think of a better place to put it), and now I'm translating the query param into an epoch date string in the NotehubDataProvider file.

I still have to call the same function in this file though in case we're falling back to the `HUB_HISTORICAL_DATA_RECENT_MINUTES` env var